storage/innobase/xtrabackup/src/redo_log.h

```diff
@@ -353,6 +354,12 @@ class Redo_Log_Data_Manager {
   /** error flag. */
   std::atomic<bool> error;
 
+  /** redo log consumer */
+  Redo_Log_Consumer redo_log_consumer;
+
+  /** MySQL connection to register redo log consumer */
+  MYSQL *redo_log_consumer_cnx = nullptr;
```

Comment:
can't we use existing mysql connection?

Reply:
@altmannmarcelo ^

Reply:
@satya-bodapati since this will happen very often (on each redo scan) and on a dedicated thread, in order to be thread safe, we would require to have a mutex on the original connection so it could be shared between multiple threads. I thought it would be safe to just open a new dedicated connection for it.

Reply:
@altmannmarcelo ok to keep it as is. Thanks for the explanation.